finish/pom.xml

```diff
@@ -138,11 +136,11 @@
             <plugin>
                 <groupId>net.wasdev.wlp.maven.plugins</groupId>
                 <artifactId>liberty-maven-plugin</artifactId>
-                <version>2.4</version>
+                <version>2.5</version>
                 <configuration>
                     <assemblyArtifact>
                         <groupId>io.openliberty</groupId>
-                        <artifactId>openliberty-runtime</artifactId>
```

Comment:
This should be left at runtime

Reply:
There are discussions about this change in the other PR too, we may on hold this one